Makefile

```diff
@@ -0,0 +1,15 @@
+.PHONY: fmt
+fmt:
+	@echo go fmt
+	go fmt $$(go list ./...)
+
+.PHONY: test
+test:
+	@echo go test
+	go test ./... -v
+
+.PHONY: mockgen
+mockgen:
+	@echo ....... Generating mock .......
+	~/go/bin/mockgen -source=interface.go -destination=interface_mock.go -package=helmclient
```

Comment:
Instead using a `Makefile`, a [`//go:generate` annotation](https://blog.golang.org/generate) might be the more "Go" way for code generation.

Also, I'd just assume that `mockgen` is in `$PATH`, whereever it is (so just `mockgen` instead of `~/go/bin/mockgen`, or at the very least, `$GOPATH/bin/mockgen`).